Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -783,6 +788,20 @@ namespace Remotion.BocAutoCompleteReferenceValue
             return true;
         }
 
+        function requestMoreData(term: string, offset: number): Promise<BocAutoCompleteReferenceValueCacheableSearchResult> {
+            return new Promise((resolve, reject) => {
+                const successHandler = (term: string, result: BocAutoCompleteReferenceValueCacheableSearchResult) => {
+                    resolve(result);
+                };
+                const failureHandler = (error: string) => {
+                    select.hide();
+                    reject();
+                };
```

Comment:
Is this triggering the error popup? (this is what would be the expected behavior as a user)

Reply:
Yes, it is.